REBASE.openshift.md

```diff
@@ -408,3 +408,110 @@ git log v1.21.1..v1.21.2 --ancestry-path --reverse --no-merges
 6. Update openshift dependencies and re-run `hack/update-vendor.sh`.
 7. Update kubernetes version in `openshift-hack/images/hyperkube/Dockerfile.rhel`.
 8. Run `make update` see [Updating generated files](#updating-generated-files).
+
+##  Z stream release
+
+1. Clone the official `kubernetes` repository in your system:
+
+```
+git clone https://github.com/kubernetes/kubernetes
+```
+
+2. Add the remote `openshift`:
+
+```
+git remote add openshift https://github.com/openshift/kubernetes
+```
+
+3. Fetch data from the remote `openshift`:
+
+```
+git fetch openshift
+```
+
+4. Checkout to release-Y.Z branch
+
+An example to checkout to release 4.9 branch:
+
+```
+git checkout openshift/release-4.9
+```
+
+5. For z stream release of Kubernetes `x.y`, select the highest patch version of `x.y` release from https://github.com/kubernetes/kubernetes/releases i.e. `vx.y.z` where z is the highest patch version.
+
+Merge the latest kubernetes version with the command:
+
+```
+git merge <version>
+```
+Example:
+
+```
+git merge v1.21.4
+````
+
+6. Create a commit **with the conflicts** using the following command:
+
+```
+git commit -m "UPSTREAM: <drop>: manually resolve conflicts"
+```
+
+7. Resolve the conflicts manually. In most cases, pick the latest version of dependencies.
```

Comment:
Resolve conflicts should be separate commit, like so: https://github.com/openshift/kubernetes/pull/916/commits/78acef811bafc2a96df867e2a060a11eb6c81225 with `UPSTREAM: <drop>: manually resolve conflicts` to show reviewers that this is the most critical part to look after.